tests/subscriptions/test_codecs.py

```diff
@@ -70,247 +72,248 @@ def build_snql_subscription_data(
     ),
 ]
 
-
-def assert_entity_subscription_on_subscription_class(
-    organization: Optional[int],
-    subscription: SubscriptionData,
-    entity_key: EntityKey,
-) -> None:
-    subscription_cls = get_entity_subscription(entity_key)
-    if organization:
-        assert isinstance(subscription.entity_subscription, subscription_cls)
-        assert getattr(subscription.entity_subscription, "organization") == organization
-    else:
-        assert isinstance(subscription.entity_subscription, EventsSubscription)
-        with pytest.raises(AttributeError):
-            getattr(subscription.entity_subscription, "organization")
-
-
-@pytest.mark.parametrize("builder, organization, entity_key", SNQL_CASES)
-def test_basic(
```

Comment:
I don't want to read through all this: you didn't actually remove any tests right? Just moved them around?

Reply:
Yup, wrapped everything inside `ConfigurationTest` to reset data factory